config/routes.rb

```diff
@@ -92,6 +92,12 @@
     get "tags/:tag", to: "projects#get_projects", as: "tag"
   end
 
+  # lti
+  get 'lti/launch'
+  post 'lti/launch'
+  get 'lti/submitscore'
+  post 'lti/submitscore'
```

Comment:
I think you can use rails scopes here. 
https://devblast.com/b/rails-5-routes-scope-vs-namespace

Reply:
done